### PR TITLE
WIP: docs(terms and conditions): add

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrum/gui/qt/terms_and_conditions_mixin.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+class WarningDialog(WindowModalDialog):
+    def __init__(self, parent, title=_('Warning')):
+        super().__init__(parent=parent, title=title)
+        self.second_chance_for_reading = True
+        vbox = QVBoxLayout(self)
+        warning_box = QHBoxLayout()
+        # FIXME new line works only with <br> not \n
+        warning_label = QLabel(
+            '<b>' + _('Are you sure?') + '</b>' + '<br>' +
+            _("Note that if you disagree to our Terms & Conditions you won't be able to use the Electrum Vault application.")
```
Comment (PJLewandowski) [inline electrum/gui/qt/terms_and_conditions_mixin.py:30]: Electrum Cash

Comment (kj-crypto) [reply electrum/gui/qt/terms_and_conditions_mixin.py]: Name changed into `Elcash Wallet`

**Review by PJLewandowski — APPROVED** [review]

File: electrum/terms_and_conditions/en_UK.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
...t quia fugiat minus eum! Iusto eaque sapiente laboriosam aperiam itaque mollitia sed, reiciendis et odit officiis eius necessitatibus ea numquam minima ut voluptatum praesentium consectetur dolorum explicabo omnis optio debitis. Officia quos aspernatur quae ducimus minus reprehenderit corporis inventore rerum, temporibus culpa aperiam praesentium suscipit quam autem assumenda reiciendis, odio voluptate sapiente, beatae dolorem! Suscipit nam pariatur unde similique fuga enim ducimus, modi harum animi alias quam sint, repellendus labore, ullam eaque corporis veritatis soluta illo! Eius explicabo rem mollitia quibusdam eos molestiae eveniet alias eum hic porro sint distinctio, facilis ipsam doloremque eligendi doloribus blanditiis deserunt dignissimos sed, totam maxime praesentium iusto. Quasi aliquam nemo quo expedita, obcaecati deleniti vel minima, earum soluta eligendi eveniet debitis iste nulla excepturi, consequatur commodi! Iure, beatae temporibus magni error cupiditate facilis asperiores. Odio suscipit, excepturi similique distinctio, consectetur cumque obcaecati natus consequuntur at explicabo quod dolores recusandae voluptatum hic magni repellat asperiores rerum quibusdam in!
```
Comment (picq) [inline electrum/terms_and_conditions/en_UK.html:8]: As I see it, we need to wait for proper content here

Comment (kj-crypto) [reply electrum/terms_and_conditions/en_UK.html:8]: done

**Pull request closed without merging**